azure-webapp-maven-plugin/src/test/resources/pom-linux.xml

```diff
@@ -25,6 +25,7 @@
                             <value>80</value>
                         </property>
                     </appSettings>
+                    <deploymentType>auto</deploymentType>

```

Comment:
Why we need explicitly set deployment type to auto here?

Reply:
testing purpose